packages/http/libraries/core/src/http/calculations/requestParamFactory.ts

```diff
@@ -0,0 +1,38 @@
+import { Newable } from 'inversify';
+
+import { ControllerMethodParameterMetadata } from '../../routerExplorer/model/ControllerMethodParameterMetadata';
+import { CustomParameterDecoratorHandler } from '../models/CustomParameterDecoratorHandler';
+import { RequestMethodParameterType } from '../models/RequestMethodParameterType';
+import { Pipe } from '../pipe/model/Pipe';
+import { requestParam } from './requestParam';
+
+export function requestParamFactory(
```

Comment:
```suggestion
export function buildRequestParameterDecorator(
```
What do you think about this alternative name?